### PR TITLE
[Swift] use stdlib (single pass)

### PR DESCRIPTION
**Pull request by Dante-Broggi**
The first commit requires a minimum Swift version of Swift 5

### CODE REVIEW

File: runtime/Swift/Sources/Antlr4/misc/BitSet.swift [hunk 1]
```diff
@@ -645,41 +645,7 @@ public class BitSet: Hashable, CustomStringConvertible {
...
-            n = n - 4
-            x = y
-        }
-        y = x << 2
-        if y != 0 {
-            n = n - 2
-            x = y
-        }
-        return Int(n - ((x << 1) >>> 31))
+        return i.trailingZeroBitCount
```
Comment (KvanTTT) [inline runtime/Swift/Sources/Antlr4/misc/BitSet.swift:648]: Is containing function required at all? `numberOfTrailingZeros` can be replaced with `trailingZeroBitCount`

File: runtime/Swift/Sources/Antlr4/misc/BitSet.swift [hunk 2]
```diff
@@ -814,35 +780,7 @@ public class BitSet: Hashable, CustomStringConvertible {
...
-            x <<= 4
-        }
-        if x >>> 30 == 0 {
-            n += 2
-            x <<= 2
-        }
-        n -= x >>> 31
-
-        return Int(n)
+        return i.leadingZeroBitCount
```
Comment (KvanTTT) [inline runtime/Swift/Sources/Antlr4/misc/BitSet.swift:783]: `numberOfLeadingZeros` -> `leadingZeroBitCount` for all usings.

File: runtime/Swift/Sources/Antlr4/misc/BitSet.swift [hunk 3]
```diff
@@ -903,16 +841,7 @@ public class BitSet: Hashable, CustomStringConvertible {
...
-        // HD, Figure 5-14
-        i = i - ((i >>> 1) & 0x5555555555555555)
-        i = (i & 0x3333333333333333) + ((i >>> 2) & 0x3333333333333333)
-        i = (i + (i >>> 4)) & 0x0f0f0f0f0f0f0f0f
-        i = i + (i >>> 8)
-        i = i + (i >>> 16)
-        i = i + (i >>> 32)
-
-        return Int(i) & 0x7f
+        return i.nonzeroBitCount
```
Comment (KvanTTT) [inline runtime/Swift/Sources/Antlr4/misc/BitSet.swift:844]: `bitCount` -> `nonzeroBitCount` for all usings.

File: runtime/Swift/Sources/Antlr4/misc/BitSet.swift (hunk 1 above)
Comment (Dante-Broggi) [reply runtime/Swift/Sources/Antlr4/misc/BitSet.swift:648]: I have not yet changed any public API (which these functions are) but as you are requesting it, I will inline these and remove them.

Comment (KvanTTT) [reply runtime/Swift/Sources/Antlr4/misc/BitSet.swift]: Anyway we've changed a lot of public API in 4.10 and removed outdated function. Awaiting @parrt for the final decision.

**Comment by parrt** [comment]
hi. yeah, maybe we can revisit after 4.10? changed the ATNDeserialization a lot!

File: runtime/Swift/Sources/Antlr4/misc/BitSet.swift (hunk 1 above)
Comment (parrt) [reply runtime/Swift/Sources/Antlr4/misc/BitSet.swift]: What duplications are we talking about? Sorry I'm trying to review lots of different things.

Comment (KvanTTT) [reply runtime/Swift/Sources/Antlr4/misc/BitSet.swift]: Nevermind, @Dante-Broggi has already removed useless public API functions.

**Pull request merged**